### PR TITLE
github: workflows: add deploy workflow

### PR DESCRIPTION
**Pull request by paivin-dn**
## Motivation
We need to have a workflow that can update a version in our service manifests.

## Solution
The `deploy.yaml` workflow has been added. This workflow updates a version in a manifest and pushes this change to a repository as `EmlidBuilderBot`.

## Checklist

<details open=true>
  <summary>PR checklist (self-check before review)</summary>

- [X] The branch name fits the `pr-<prefix>-<task-name>` pattern;
- [X] The PR has a meaningful name using the `<prefix>: <description>` pattern;
- [X] The PR is properly labeled depending on the change;
- [X] The PR is assigned to you;
- [X] If the PR is based on another PR, links to them are added to the PR header and the "has PR dependency" label is set;
- [X] Self-review. Review your code, commits, and the PR description;
- [X] Make sure that you updated the corresponding docs in the `docs/` directory;
- [X] Links to the updated docs are added to the PR header (if any);
- [X] CI successfully finished;

</details>

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          ref: ${{ inputs.branch }}
+          token: ${{ secrets.gh_token }}
+      - name: Set up new dev versions
+        run: |
+          yq e -i ".image.tag = \"${{ inputs.version }}\"" values/${{ inputs.values_folder }}/${{ inputs.service }}.yaml
+      - name: Commit and push changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "${{ inputs.service }}: ${{ inputs.version }}"
+          commit_user_email: "<builderbot@emlid.com>"
```
Comment (ivnsmirnov) [inline .github/workflows/deploy.yaml:51]: fix the email

Comment (paivin-dn) [reply .github/workflows/deploy.yaml]: Fixed

**Review by ivnsmirnov — APPROVED** [review]
GJ!

File: .github/workflows/deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        required: true
+        type: string
+        default: "myorg/myrepo"
+      branch:
+        description: "Repository branch"
+        required: false
+        type: string
+        default: "master"
+      values_folder:
+        description: "A folder is contained helm values files"
```
Comment (anastasiia-garnaga) [inline .github/workflows/deploy.yaml:17]: Nit:
```suggestion
        description: "A folder containing helm values files"
```
?

**Review by anastasiia-garnaga — APPROVED** [review]
LGTM!

File: .github/workflows/deploy.yaml (hunk 2 above)
Comment (paivin-dn) [reply .github/workflows/deploy.yaml]: Helm values files folder =)

**Review by glebignatieff — APPROVED** [review]
GJ!

**Pull request merged**